pkg/cloud-provider/cloudapi/aws/aws_ec2.go

```diff
@@ -333,3 +348,55 @@ func (ec2Cfg *ec2ServiceConfig) buildMapVpcPeers() (map[string][]string, error)
 	}
 	return vpcPeers, nil
 }
+
+func (ec2Cfg *ec2ServiceConfig) NeedPollWithoutFilter() bool {
+	return true
```

Comment:
Comment on the function. Also, it seems dummy function without any logic ?

Reply:
Already there is a comment in services_common.go. 
I am using a variable pollVpc at service config level(set to true during service config creation) to force inventory polling for ec2 service.